packages/smooth_app/lib/pages/prices/price_model.dart

```diff
@@ -50,6 +52,14 @@ class PriceModel with ChangeNotifier {
     return false;
   }
 
+  late bool _multipleProducts;
+  bool showAddProductCard() {
+    if (_multipleProducts == true) {
+      return true;
+    }
+    return false;
+  }
+
```

Comment:
```suggestion
  /// "Should we support multiple products?" (instead of a single)
  final bool multipleProducts;
```

For the record, your previous syntax was a bit heavy:
```dart
  bool showAddProductCard() {
    if (_multipleProducts == true) {
      return true;
    }
    return false;
  }
```
The following would have been better:
```dart
  bool get showAddProductCard => _multipleProducts;
```